contracts/cw-core/src/state.rs

```diff
@@ -61,3 +61,6 @@ pub const CW20_LIST: Map<Addr, Empty> = Map::new("cw20s");
 /// Set of cw721 tokens that have been registered with this contract's
 /// treasury.
 pub const CW721_LIST: Map<Addr, Empty> = Map::new("cw721s");
+
+/// List of SubDAOs associated to this DAO
+pub const SUBDAO_LIST: Map<&Addr, Option<String>> = Map::new("sub_daos");
```

Comment:
Little comment explaining what this field is for:

```suggestion
/// List of SubDAOs associated to this DAO. Each SubDAO has an optional charter.
pub const SUBDAO_LIST: Map<&Addr, Option<String>> = Map::new("sub_daos");
```